composer.json

```diff
@@ -24,7 +24,8 @@
     },
     "autoload-dev": {
         "psr-4": {
-            "League\\Flysystem\\AwsS3v3\\Stub\\": "stub"
+            "League\\Flysystem\\AwsS3v3\\Stub\\": "stub",
+            "spec\\League\\Flysystem\\AwsS3v3\\": "spec"
```

Comment:
Not sure about the CI thought this would fix it but maybe its a bug with: https://github.com/phpspec/phpspec/issues/1239